src/modality-agnostic-files/phenotypic-and-assessment-data.md

```diff
@@ -80,3 +88,56 @@ descriptions with a `Derivative` field that, when set to true, indicates that
 values in the corresponding column is a transformation of values from other
 columns (for example a summary score based on a subset of items in a
 questionnaire).
+
+## Additional validation
+
+When the [`AdditionalValidation` key](dataset-description.md#additional-validation)
+contains `"Phenotype"` in the `dataset_description.json`,
+the following expectations apply to phenotypic and assessment data.
+
+1.  It is REQUIRED to aggregate all participant data into
+    one TSV per tabular phenotypic file.
+
+1.  Each tabular phenotypic data TSV file MUST be accompanied by
+    a corresponding data dictionary JSON file.
+
+1.  Whenever possible, it is RECOMMENDED to add `MeasurementToolMetadata` to
+    each `phenotype/<measurement_tool_name>.json` data dictionary.
+    This improves reusability and provides clarity about the measurement tool.
+
+1.  Each measurement tool SHOULD have an independent
+    aggregated data TSV file in which the user collects all subjects, sessions,
+    and/or runs of data as one entry per row (with a row defined by
+    the smallest unit of acquisition). In other words:
+
+    1.  Each row MUST start with `participant_id`.
+
+    1.  Each TSV file MUST contain a `session_id` column when
+        multiple [sessions](../glossary.md#session-entities) are present
+        in the data set regardless of whether those sessions are in
+        the `phenotype/` data, `sub-<label>/` data, or a combination of the two.
```

Comment:
How about linking an example dataset that demonstrates the following two cases: 
1. `phenotype` and `imaging` data were collected in the same session, and therefore have the same `session_id`. 
2. `phenotype` and `imaging` data were collected in two separate sessions, and therefore, `phenotype` and `sub-<label>` data have different `session_id`.

Reply:
Attempted to resolve in the next commit.